src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java

```diff
@@ -105,4 +96,13 @@ public RequestFormat getRequestFormat() {
         }
         return null;
     }
+
+    @JsonIgnore
+    public boolean isRefused() {

```

Comment:
Is the REFUSE value always "refuse"? If so, could we turn this into a one liner and just use equals("refuse") on ccdDefinition?

Fine if not, but could reduce some lines.